### PR TITLE
modify ooi-ctdmo-ghqr-decode.xml for ticket 12191 ( add paser to pars…

### PR DESCRIPTION
**Pull request by vipullakhani**
…e CTDMO recovered_host instrument data (ctdmo_ghqr) )

### CODE REVIEW

**Comment by danmergens** [comment]
looks good, but I'd like Rene to review before we merge

**Comment by vipullakhani** [comment]
ok,
This morning i had very brief lesson learned from him,
and i was missing couple of key part, in order to run uframe-test.

This ooi-ctdmo-ghqr-decode.xml was one of these.

Thanks,
vipul

File: res/spring/ooi-ctdmo-ghqr-sio-decode.xml [hunk 1]
```diff
@@ -9,6 +9,12 @@
         <constructor-arg index="2" value="mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_co_recovered_driver" />
     </bean>
 
+    <bean id="ctdmo-ghqr-sio-ctRecoveredProcessor" class="com.raytheon.uf.edex.ooi.decoder.dataset.ctdmo_ghqr.CtdmoGhqrCoFileDecoder">
```
Comment (renegelinas) [inline res/spring/ooi-ctdmo-ghqr-sio-decode.xml]: **ctdmo-ghqr-sio-ctRecoveredProcessor** should be **ctdmo-ghqr-sio_ctRecoveredProcessor** to match the "bean ref=" in the new route block below.

**Comment by renegelinas** [comment]
Everything looks good, except in the bean id, the third "-" should be a "_".

File: .idea/dictionaries/admin.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
```
Comment (danmergens) [inline .idea/dictionaries/admin.xml]: remove this file from the commit

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (danmergens) [inline .idea/vcs.xml]: remove this file from the commit

**Pull request merged**